qt/python/mantidqt/mantidqt/widgets/sliceviewer/presenters/presenter.py

```diff
@@ -226,6 +227,11 @@ def dimensions_changed(self):
             self.new_plot()
         self._call_cutviewer_presenter_if_created("on_dimension_changed")
 
+    def extents_changed(self):
```

Comment:
Maybe this needs to be called inside `dimensions changed` to fix the issue with the limits being applied before the rebin when transposing? However, ultimately that just makes a new plot again which is a bit wasteful.

Reply:
That seems to do the trick.

Reply:
All we have to do is update the axis limits when transposing. That way we don't have to make a new plot